front/app/containers/Authentication/steps/_components/SSOVerificationButton/index.tsx

```diff
@@ -67,6 +67,10 @@ const SSOVerificationButton = ({
       label: messages.verifyKeycloak,
       icon: 'idporten',
     },
+    twoday: {
+      label: messages.verifyTwoday,
+      icon: 'bullseye', // TODO: this
+    },
```

Comment:
Is this TODO relevant still?

Reply:
Good catch, just changed this to the BankID icon